src/Dapr.Actors/Runtime/ActorMethodContext.cs

```diff
@@ -40,6 +47,18 @@ public ActorCallType CallType
             get { return this.actorCallType; }
         }
 
+        /// <summary>
+        /// Gets the callback method to be called by actor runtime (e.g. actor interface method, timer callback etc.).
+        /// </summary>
+        /// <value>
+        /// An <see cref="Callback"/> representing the call type.
+        /// </value>
+        public MethodInfo Callback
+        {
+            get { return this.callback; }
+        }
+
+
```

Comment:
Are the changes to this file used? This seems like its from an earlier iteration